Meshery-Adapters/consul-meshery-adapter/step1.md

```diff
@@ -14,9 +14,9 @@ Meshery can be downloaded, installed, and launched with a single command:
 
 Meshery is now available at host:`9081`.
 
-Find the **Meshery UI** tab is configured to be listening at `http://localhost:9081`/
+Find the **Meshery Server** tab is configured to be listening at `http://localhost:9081`/
```

Comment:
Please revert this change back we still want the instruction to state Meshery UI. The tab name should be changed to Meshery UI, which is a different issue to work on.

Reply:
ok @asubedy